src/Model/AbstractArticle.php

```diff
@@ -285,11 +285,16 @@ public function setValidatedAt(\DateTimeInterface $validatedAt = null): void
         $this->validatedAt = $validatedAt;
     }
 
-    public function getValidatedAt(): \DateTimeInterface
+    public function getValidatedAt(): ?\DateTimeInterface
```

Comment:
Might be a problem with how the whole project is working right now, for instance if entities are used directly in forms, it will be an issue I think. So I'll just approve the PR.